src/Microsoft.VisualStudio.SlowCheetah/Build/Microsoft.VisualStudio.SlowCheetah.targets

```diff
@@ -58,18 +66,41 @@ Copyright (C) Sayed Ibrahim Hashimi, Chuck England 2011-2013. All rights reserve
     <Message Text="SlowCheetah: Collecting Transform files" Importance="low"/>
 
     <ItemGroup>
-      <ScFilesToTransform Include="@(None);@(Content);@(Resource);@(EmbeddedResource)"
-                         Condition=" '%(TransformOnBuild)' == 'true' ">
+      <_ScNoneFilesToTransform Include="@(_NoneWithTargetPath)" Condition="'%(TransformOnBuild)' == 'true'">
+        <BuildAction>None</BuildAction>
+      </_ScNoneFilesToTransform>
+
+      <_ScContentFilesToTransform Include="@(ContentWithTargetPath)" Condition="'%(TransformOnBuild)' == 'true'">
```

Comment:
There is no need for multiple item types at this point. Include them directly into `ScFilesToTransform` differentiate them via the `BuildAction` later. This will require multiple calls to adding to `ScFilesToTransform`, but you can perform the 4 adds with just the `BuildAction` metadata assignment, then update the entire item group at once for all of the additional metadata.